Source/Scene/GltfVertexBufferLoader.js

```diff
@@ -384,7 +384,11 @@ function loadFromDraco(vertexBufferLoader) {
     }
 
     // Now wait for process() to run to finish loading
-    vertexBufferLoader._typedArray = typedArray;
+    vertexBufferLoader._typedArray = new Uint8Array(
+      typedArray.buffer,
+      typedArray.byteOffset,
+      typedArray.byteLength
+    );
```

Comment:
Before this fix `_typedArray` was not actually a `Uint8Array`.

This and https://github.com/CesiumGS/cesium/pull/10541/files#r919520142 were causing Draco attributes to fail when packed.